### ISSUE TITLE
Rule: button-name changes

### ISSUE THREAD

**Issue by jkodu** [dequelabs/axe-core#1139]
Should fail: `aria-hidden` buttons should be tested.

```html
<button aria-hidden="true"></button>
```

**Comment by jkodu** [comment]
Discussion to correct the rule - https://github.com/auto-wcag/auto-wcag/issues/264
